### PR TITLE
Fix release notes to match schema

### PR DESCRIPTION
**Pull request by brian-avery**
Preview of the release notes (additional content added that's not in 1.7). GitHub doesn't accept HTML, so I added a .txt extension. 
[minorReleaseNotes.md.html.txt](https://github.com/istio/istio/files/5124354/minorReleaseNotes.md.html.txt)

Preview of the upgrade notes:
[upgradeNotes.md.html.txt](https://github.com/istio/istio/files/5124361/upgradeNotes.md.html.txt)

This PR fixes the release notes to match the release notes schema.

Pull Request Attributes

Please check any characteristics that apply to this pull request. 

[x] Does not have any changes that may affect Istio users.

### CODE REVIEW

File: releasenotes/notes/k8s-auth.yaml [hunk 1]
```diff
@@ -1,7 +1,8 @@
 apiVersion: release-notes/v2
 kind: feature
 area: security
-releaseNotes: |
-       Support migration and concurrent use of regular K8S tokens as well as new K8S tokens with audience. This feature is enabled by 
+releaseNotes:
+- |
+       Support migration and concurrent use of regular K8S tokens as well as new K8S tokens with audience. This feature is enabled by
```
Comment (shamsher31) [inline releasenotes/notes/k8s-auth.yaml:6]: Are the spaces required here? Other notes don't have large spaces.

Comment (shamsher31) [reply releasenotes/notes/k8s-auth.yaml:6]: Also is it ok to append "**Added**" for security notes?

Comment (brian-avery) [reply releasenotes/notes/k8s-auth.yaml]: Thanks. Removed the spaces. Went through and looked at the content to make sure that all release notes currently display with the addition of support for action as well.

Comment (brian-avery) [reply releasenotes/notes/k8s-auth.yaml]: The note above is a release note from the security section and should follow the release notes. Vulnerability notes would appear in the securityNotes section and have their own format: https://github.com/istio/istio/blob/master/releasenotes/README.md#security-notes

**Comment by brian-avery** [comment]
Release notes: 

This release contains bug fixes to improve robustness. This release note describes what’s different between Istio 1.6.2 and Istio 1.6.3.

{{< relnote >}}

# Changes

- **Improved** `istioctl analyze` to find the exact line number with configuration errors when analyzing yaml files.
Before, it would return the first line of the resource with the error.
 ([Issue #22872](https://github.com/istio/istio/issues/22872))

- **Updated** the debug handlers on HTTP are now conditionally added based environment variable ENABLE_DEBUG_ON_HTTP.
Setting "ENABLE_DEBUG_ON_HTTP" to false will not add debug handlers on HTTP ports and is recommended setting for
production. In future releases, they will be replaced with "XDS-over-TLS" instead of exposing them on HTTP interface.
 

- **Updated** the Envoy filter names that control plane generates to meet the canonical naming standard. See [Envoy's deprecation policy](https://www.envoyproxy.io/docs/envoy/latest/version_history/v1.14.0#deprecated)
 

- **Updated** the "Control Plane Dashboard" and the "Performance Dashboard" to use the `container_memory_working_set_bytes` metric
to display memory. This metric only counts memory that *cannot be reclaimed* by the kernel even under memory pressure,
and therefore more relevant for tracking. It is also consistent with `kubectl top`. The reported values are lower than
the previous values.
 

- **Added** support for injecting istio-cni into `k8s.v1.cni.cncf.io/networks` annotation with pre-existing value that uses JSON notation.
 ([Issue #25744](https://github.com/istio/istio/issues/25744))

- **Added** Update Istio Workload and Istio Service dashboards to improve loading time.
 ([Issue #22408](https://github.com/istio/istio/issues/22408))
- **Added** parameterise Grafana dashboards with datasource
 ([Issue #22408](https://github.com/istio/istio/issues/22408))

- **Added** support for client side Envoy secure naming config when trust domain alias is used.
Fix the multi cluster service discovery client SAN generation: takes all endpoints' service accounts
into account, rather than the first found service registry.
 

- **Added** action 'AUDIT' to Authorization Policy that can be used to determine which requests should be audited.
 ([Issue #25591](https://github.com/istio/istio/issues/25591))

- **Added** experimental caching for endpoints, to support large scale clusters. This feature is disabled by default and can
be enabled by setting the `PILOT_ENABLE_EDS_CACHE` environment variable in Istiod.
 

- **Added** support for migration and concurrent use of regular K8S tokens as well as new K8S tokens with audience. This feature is enabled by
default, can be disabled by REQUIRE_3P_TOKEN environment variable in Istiod, which will require new tokens with audience. The
TOKEN_AUDIENCE environment variable allows customizing the checked audience, default remains istio-ca.
 

- **Added** port 15012 to the default list of ports for the `istio-ingressgateway` Service.
 ([Issue #25933](https://github.com/istio/istio/issues/25933))

- **Added** Istio 1.8 supports kubernetes versions 1.17 to 1.19.
 ([Issue #25793](https://github.com/istio/istio/issues/25793))

- **Deprecated** installation flags `values.global.meshExpansion.enabled` in favor of user-managed config and `values.gateways.istio-ingressgateway.meshExpansionPorts` in favor of `components.ingressGateways[name=istio-ingressgateway].k8s.service.ports`
 ([Issue #25933](https://github.com/istio/istio/issues/25933))


- **Fixed** remove endpoints when the new labels in `WorkloadEntry` do not match the `workloadSelector` in `ServiceEntry`.
 ([Issue #25678](https://github.com/istio/istio/issues/25678))

- **Fixed** Remove unreachable endpoints for non-injected workloads across networks.
 ([Issue #26517](https://github.com/istio/istio/issues/26517))

- **Fixed** headless services endpoints update will not trigger any xds pushes for sidecar proxies
 ([Issue #26617](https://github.com/istio/istio/issues/26617))


- **Removed** all Mixer-related features and functionality. This is a scheduled
removal of a deprecated Istio services and deployments, as well as
Mixer-focused CRDs and component and related functionality.
 ([Issue #25333](https://github.com/istio/istio/issues/25333)),([Issue #24300](https://github.com/istio/istio/issues/24300))

- **Removed** compiled in support for Consul service registry. Integration will be done using XDS in the future.
 

- **Removed** the installation of telemetry addons (Prometheus, Grafana, Zipkin, Jaeger, Kiali) from installation by `istioctl`. See [Reworking our Addon Integrations](/blog/2020/addon-rework/) for more info.
 ([Issue #23868](https://github.com/istio/istio/issues/23868)),([Issue #23583](https://github.com/istio/istio/issues/23583))
- **Removed** istio-telemetry and istio-policy from installation by `istioctl`.
 ([Issue #23868](https://github.com/istio/istio/issues/23868)),([Issue #23583](https://github.com/istio/istio/issues/23583))



# Security update

**Comment by brian-avery** [comment]
Upgrade notes: 

When you upgrade from Istio 1.5.x to Istio 1.6.x, you need to consider the changes on this page.
These notes detail the changes which purposefully break backwards compatibility with Istio 1.5.x.
The notes also mention changes which preserve backwards compatibility while introducing new behavior.
Changes are only included if the new behavior would be unexpected to a user of Istio 1.5.x.

## Mixer is no longer supported in Istio.
If you are using the `istio-policy` or `istio-telemetry` services, or any
related Mixer configuration, you will not be able to upgrade without taking
action to either (a) convert your existing configuration and code to the new
extension model for Istio or (b) using the gRPC shim developed to bridge
transition to the new model. For more details, please refer the docs on
istio.io.

## Use the new filter names for EnvoyFilter
If you are using EnvoyFilter API, it is recommended to change to the new filter names https://www.envoyproxy.io/docs/envoy/latest/version_history/v1.14.0#deprecated.
The deprecated filter names will be supported in this release for backward compatibility but will be removed in future releases.

## Avoid use of mesh expansion installation flags
To ease setup for multicluster and virtual machines while giving more control to users, the `meshExpansion` and `meshExpansionPorts` installation flags have been deprecated, and port 15012 has been added to the default list of ports for the `istio-ingressgateway` Service.

For users with `values.global.meshExpansion.enabled=true`, perform the following steps before upgrading Istio:

1. Apply the code sample for exposing Istiod through ingress.

   {{< text bash >}}
   $ kubectl apply -f @samples/istiod-gateway/istiod-gateway.yaml@
   {{< /text >}}

   This removes `operator.istio.io/managed` labels from the associated Istio networking resources so that the Istio installer won't delete them. After this step, you can modify these resources freely.

1. If `components.ingressGateways[name=istio-ingressgateway].k8s.service.ports` is overridden, add port 15012 to the list of ports:

   {{< text yaml >}}
        - port: 15012
          targetPort: 15012
          name: tcp-istiod
   {{< /text >}}

1. If `values.gateways.istio-ingressgateway.meshExpansionPorts` is set, move all ports to `components.ingressGateways[name=istio-ingressgateway].k8s.service.ports` if they're not already present. Then, unset this value.

1. Unset `values.global.meshExpansion.enabled`.

**Review by shamsher31 — APPROVED** [review]
LGTM, Thanks @brian-avery.

**Pull request merged**